ros_buildfarm/templates/status/js/setup.js

```diff
@@ -6,7 +6,8 @@ var QUERY_TRANSFORMS = {
   'GRAY': '<a class="i"></a>',
   'RED1': '<td><a class="m"></a>',
   'RED2': '</a><a class="m"></a><a',
-  'RED3': '<a class="m"></a></td>'
+  'RED3': '<a class="m"></a></td>',
+  'UNMAINTAINED' : '<span class="unmaintained"'
```

Comment:
The query could just use the lower case search "unmaintained". I don't expect that token to appear in other parts of the status page beside the status column. Therefore this change is not necessary.